lab-cortney/package.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
This is a great start, add information into all the fields (author, scripts>test -> mocha, etc).  Deducting a point for the missing information.